Composer/packages/client/src/recoilModel/dispatchers/qna.ts

```diff
@@ -50,7 +50,9 @@ export const createQnAFileState = async (
 ) => {
   const { set, snapshot } = callbackHelpers;
   const qnaFiles = await snapshot.getPromise(qnaFilesState(projectId));
-  const locale = await snapshot.getPromise(localeState(projectId));
+  //const locale = await snapshot.getPromise(localeState(projectId));
+  //To do: support other languages
+  const locale = 'en-us';
```

Comment:
For other methods in this dispatcher should also use fixed locale en-us.
`removeQnAFileState`
`renameKBFileState`
`updateQnAFileState`

Reply:
solved. And it seems createQnAKBFromScratch and createQnAKBFromUrl should use fixed locale en-us